tests/test_unit/test_annotations/test_validators.py

```diff
@@ -0,0 +1,408 @@
+    """
+    return {
+        "type": "object",
+        "properties": {
+            "a": {
+                "type": "array",
+                "items": {"type": "string"},
+            },
+            "b": {
+                "type": "object",
+                "properties": {"b1": {"type": "string"}},
+            },
+            "c": {
+                "type": "object",
+                "properties": {
+                    "c1": {"type": "string"},
+                    "c2": {"type": "string"},
+                },
+            },
+        },
+    }
+
+
+@pytest.fixture()
+def default_VIA_schema() -> dict:
+    """Get default VIA schema."""
+    from ethology.annotations.json_schemas.utils import _get_default_VIA_schema
+
+    return _get_default_VIA_schema()
+
+
+@pytest.fixture()
+def default_COCO_schema() -> dict:
+    """Get default COCO schema."""
+    from ethology.annotations.json_schemas.utils import (
+        _get_default_COCO_schema,
+    )
+
+    return _get_default_COCO_schema()
+
+
+@pytest.mark.parametrize(
+    "input_file, validator",
+    [
+        ("VIA_JSON_sample_1.json", ValidVIA),
+        ("VIA_JSON_sample_2.json", ValidVIA),
+        ("COCO_JSON_sample_1.json", ValidCOCO),
+        ("COCO_JSON_sample_2.json", ValidCOCO),
+    ],
+)
+def test_validators_valid_input_files(
+    input_file: str,
+    validator: type[ValidVIA | ValidCOCO],
+    annotations_test_data: dict,
+):
+    """Test the file validator with valid inputs."""
+    filepath = annotations_test_data[input_file]
+    with does_not_raise():
+        validator(path=filepath)
+
+
+@pytest.mark.parametrize(
+    "invalid_input_file, validator, expected_exception, log_message",
+    [
+        (
+            "json_file_decode_error",
+            ValidVIA,
+            pytest.raises(ValueError),
+            "Error decoding JSON data from file",
+        ),
+        (
+            "json_file_not_found_error",
+            ValidVIA,
+            pytest.raises(FileNotFoundError),
+            "No such file or directory: ",
+        ),
+        (
+            "json_file_decode_error",
+            ValidCOCO,
+            pytest.raises(ValueError),
+            "Error decoding JSON data from file",
+        ),
+        (
+            "json_file_not_found_error",
+            ValidCOCO,
+            pytest.raises(FileNotFoundError),
+            "No such file or directory: ",
+        ),
+        (
+            "VIA_file_schema_mismatch",
+            ValidVIA,
+            pytest.raises(jsonschema.exceptions.ValidationError),
+            "'49' is not of type 'integer'",
+        ),
+        (
+            "COCO_file_schema_mismatch",
+            ValidCOCO,
+            pytest.raises(jsonschema.exceptions.ValidationError),
+            "3 is not of type 'object'",
+        ),
+    ],
+)
+def test_validators_invalid_input_files(
+    invalid_input_file: str,
+    validator: type[ValidVIA | ValidCOCO],
+    expected_exception: pytest.raises,
+    log_message: str,
```

Comment:
what you are actualling checking should be called "error_message" not "log_message", write. There is no logging setup yet, and you are not checking tha logs, just the exception info.